partner/app/main.py

```diff
@@ -79,7 +79,9 @@ async def read_tilesp(
         ..., example="placement1", max_length=128, regex="^[a-zA-Z0-9]+$"
     ),
     country_code: str = Query(..., alias="country-code", example="US"),
-    region_code: str = Query(..., alias="region-code", example="NY"),
+    region_code: str = Query(
+        ..., alias="region-code", example="NY", regex="^([A-Z]{2}-[a-zA-Z0-9]{2,3})"
```

Comment:
Looks like this would not accept the example value `"NY"` which is taken from the API spec.